site/content/en/news/releases/1.4.0.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Announcing Envoy Gateway v1.4
+subtitle: Minor Update
+linktitle: Release v1.4
+description: Envoy Gateway v1.4 release announcement.
+publishdate: 2025-05-13
+release: v1.4.0
+skip_list: true
+---
+
+We are thrilled to announce the arrival of Envoy Gateway v1.4.0.
+
+This release represents a significant achievement, and we extend our heartfelt gratitude to the entire Envoy Gateway community for their contributions, dedication, and support. Your collaborative efforts have been instrumental in reaching this pivotal release.
+
+Thank you for being an integral part of this journey. We are excited to see how Envoy Gateway v1.4.0 will empower your operations and look forward to continuing our work together to drive the future of Cloud Native API Gateway.
+
+| [Release Notes][] | [Docs][docs] | [Compatibility Matrix][matrix] | [Install][] |
+|-------------------|--------------|--------------------------------|--------------|
+
+## What's New
+
+Envoy Gateway v1.4.0 introduces powerful new features across traffic management, observability, security, extensibility, and infrastructure — along with key stability improvements.
+
+---
+
+## 🚨 Breaking Changes
+
+- **Envoy Readiness Port**: A dedicated listener port (`19003`) is now used for Envoy proxy readiness.
+- **Access Log Format**: The default access log now uses the Envoy JSON formatter instead of the plain text formatter.
+- **xDS Snapshot Behavior**: Envoy Gateway now skips xDS snapshot updates if errors occur during xDS translation.
+- **Extension Manager Behavior**:
+  - With `failOpen: true`, translation errors are logged and suppressed.
+  - With `failOpen: false`, xDS updates are skipped for affected resources (no longer replaced).
+
+---
+
+## ✨ New Features
+
+### API & Traffic Management Enhancements
+
+- Dynamic backend routing via the new `DynamicResolver` type based on request hostnames.
+- Support for zone-aware routing.
+- Percentage-based request mirroring support.
+- New circuit breaker support for per-host thresholds.
+- Support for HTTP method and header-based authorization via `SecurityPolicy`.
```

Comment:
This feature belongs to "Security Enhancements"?

Reply:
yes my bad